snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs

```diff
@@ -159,18 +170,18 @@ public static void Main()
         //Furrytail
         //Peaches
         //List element has null value.
-        // </SnippetFullExample>
     }
+    // </SnippetFullExample>
 
+    // <SnippetFullListExample>
     public class FullExample
     { 
-        // <SnippetFullListExample>
         class FormattedAddresses : IEnumerable<string>
         {
             private List<string> internalList = new List<string>();
             public IEnumerator<string> GetEnumerator() => internalList.GetEnumerator();
```

Comment:
Missing namespace here too?

Reply:
The generic ones are loaded by default, just not the `System.Collections` namespaces.